### ISSUE TITLE
Smaccer NPC ID's keep changing on reload

### ISSUE THREAD

**Issue by iqcodez** [AIPTU/Smaccer#12]
Currently building a server with my friends and we wanted to use ur plugin because it works very well but we encountered a major issue which is that the ID's keep changing when restarting the server or reloading a world. I would like to ask you to add static ID's or to give the creator the ability to set the ids themselfes.

**Comment by AIPTU** [comment]
you can see the id using `/smaccer id`, no?

**Comment by iqcodez** [comment]
> you can see the id using `/smaccer id`, no?

Yes I can but that doesnt solve our problem. We are trying to use the NPC's for commands but the commands wont work if the ID's change on every reload. Could you provide help?

**Comment by AIPTU** [comment]
What's the command for exactly?

**Comment by AIPTU** [comment]
This is actually a known issue as the id will keep getting recreated when the server starts. 
I'll try to address it in the next version.

**Comment by AIPTU** [comment]
solved, https://github.com/AIPTU/Smaccer/commit/4c1ea0cf2cd596f833800b1ff41c2528e54e3ffd
